src/registrar/management/commands/load_transition_domain.py

```diff
@@ -0,0 +1,463 @@
+                        )
+                    if new_entry_domainName not in duplicate_domains:
+                        duplicate_domains.append(new_entry_domainName)
+                if tempEntry_domainUserPair is not None:
+                    if debug_on:
+                        logger.info(
+                            f"""{termColors.WARNING} DUPLICATE Verisign entries found for domain - user {termColors.BackgroundLightYellow} PAIR {termColors.ENDC}{termColors.WARNING}:  
+                            {new_entry_domainName} - {new_entry_email} {termColors.ENDC}"""  # noqa
+                        )
+                    if tempEntry_domainUserPair not in duplicate_domain_user_combos:
+                        duplicate_domain_user_combos.append(tempEntry_domainUserPair)
+                else:
+                    try:
+                        existingEntry = TransitionDomain.objects.get(
+                            username=new_entry_email, domain_name=new_entry_domainName
+                        )
+
+                        if existingEntry.status != new_entry_status:
+                            # DEBUG:
+                            if debug_on:
+                                logger.info(
+                                    f"""{termColors.OKCYAN}
+    Updating entry: {existingEntry}
+    Status: {existingEntry.status} > {new_entry_status}
+    Email Sent: {existingEntry.email_sent} > {new_entry_emailSent}
+    {termColors.ENDC}"""
+                                )
+
+                            existingEntry.status = new_entry_status
+
+                        existingEntry.email_sent = new_entry_emailSent
+                        existingEntry.save()
+                    except TransitionDomain.DoesNotExist:
+                        # no matching entry, make one
+                        newEntry = TransitionDomain(
+                            username=new_entry_email,
+                            domain_name=new_entry_domainName,
+                            status=new_entry_status,
+                            email_sent=new_entry_emailSent,
+                        )
+                        to_create.append(newEntry)
+                        total_new_entries += 1
+
+                        # DEBUG:
+                        if debug_on:
+                            logger.info(
+                                f"{termColors.OKCYAN} Adding entry {total_new_entries}: {newEntry} {termColors.ENDC}"  # noqa
+                            )
+                    except TransitionDomain.MultipleObjectsReturned:
+                        logger.info(
+                            f"""
+{termColors.FAIL}
+!!! ERROR: duplicate entries exist in the
+transtion_domain table for domain:
+{new_entry_domainName}
+----------TERMINATING----------"""
+                        )
+                        import sys
```

Comment:
**suggestion (non-blocking)**: In general I think that imports should be at the top of the file. `sys` is almost certainly already imported in the application, so we aren't saving anything by avoiding the import when this block isn't reached.

Reply:
100 percent agree with moving imports to the top f the file